lib/rules/alpha-value-notation/__tests__/index.js

```diff
@@ -271,3 +271,51 @@ testRule({
 		},
 	],
 });
+
+// testing that floating point-rounded numbers-to-percentages are handled properly
+testRule({
+	ruleName,
+	config: ['percentage'],
+	fix: true,
+
+	reject: [
```

Comment:
Let's move these tests into the `testRule`s above and remove the comment in favour of adding a `description: ..` to the test case.

We only tend to create a new `testRule` when the `config` changes or the `syntax` property is used.

Reply:
Yup, sounds good! I updated the PR, moving the test cases and adding a `description` key.